File: TIC10002/src/main/java/uff/ic/lleme/tic10002/trabalhos/s20181/Romulo_e_Lucas/src/ed/trabalho20181/TipoAssunto.java
package ed.trabalho20181;

public class TipoAssunto {
    private int tipo;
    private String titulo;
    private int urgencia;

    public int getTipo() {
        return tipo;
    }

    public void setTipo(int tipo) {
        this.tipo = tipo;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public int getUrgencia() {
        return urgencia;
    }

    public void setUrgencia(int urgencia) {
        this.urgencia = urgencia;
    }

    public TipoAssunto(int tipo, String titulo, int urgencia) {
        this.tipo = tipo;
        this.titulo = titulo;
        this.urgencia = urgencia;
    }
}

File: TIC10002/src/main/java/uff/ic/lleme/tic10002/trabalhos/s20181/GabrielDuarte_CintiaCosta/TipoAssunto.cpp
/*

	Gabriel Duarte
	Cintia Costa

*/

#ifndef TIPOASSUNTO
#define TIPOASSUNTO 1
#include <bits/stdc++.h>
using namespace std;

class TipoAssunto{
private:
	int tipo, urgencia;
	string titulo;
public:
	TipoAssunto(){}
	TipoAssunto(int _tipo, string _titulo, int _urgencia){
		tipo = _tipo;
		titulo = _titulo;
		urgencia = _urgencia;
	}
	void setTipo(int _tipo){ tipo = _tipo; }
	void setTitulo(string _titulo){ titulo = _titulo; }
	void setUrgencia(int _urgencia){ urgencia = _urgencia; }
	int getUrgencia(){ return urgencia; }
	string getTitulo(){	return titulo;	}
	int getTipo(){ return tipo; }

	void imprimirAssunto(){
		printf("Tipo: %d - Urgencia: %d - Titulo: %s\n", tipo, urgencia, titulo.c_str());
	}
};	
#endif